deserializer.go

```diff
@@ -4,9 +4,12 @@ import (
 	"bytes"
 	"encoding/gob"
 	"regexp"
+	"strconv"
+	"strings"
 	"time"
 
 	"github.com/getlantern/golog"
+	iradix "github.com/hashicorp/go-immutable-radix"
```

Comment:
Basically just because by default I think everything should be immutable to reduce overall state, and in this case post-startup there are never any changes to the tree.